experimental/adaptive-tool/src/lg/providers/completion.ts

```diff
@@ -91,6 +91,96 @@ class LGCompletionItemProvider implements vscode.CompletionItemProvider {
                 })
                 return items;
             }
+        } else if (/^> !#/.test(lineTextBefore)) {
```

Comment:
It's better to have some comments about this block to show waht does "/^> !#/.test(lineTextBefore)" mean.